core/build.gradle

```diff
@@ -137,6 +137,9 @@ dependencies {
 
     // Jetty, for testing interoperability with other servers.
     testImplementation 'org.eclipse.jetty:jetty-webapp'
+
+    // Logback
+    implementation 'ch.qos.logback:logback-classic'
```

Comment:
I think we cannot add this dependency to the `core` module because a user might want to use a different logging framework such as log4j.
So I guess we need to fork the source from the Logback and leave the copyright in the class file for that.

Reply:
I got it. I will address as you said. Thanks!